devel/external/AEGIS/document/main.tex

```diff
@@ -0,0 +1,516 @@
+\end{enumerate}
+Galaxies with the bad stamp flag in any filter are removed from the final catalogs of both filters.
+We are left with 26,517 good galaxy postage stamps in our final catalog, for both bands.
+
+\subsection{Creating the Noise Map}
+\label{sec:noise_map}
+The drizzling process introduces correlations between neighboring pixels, which artificially reduces the noise level of co-added drizzled images. Care is taken to preserve these correlations when generating noise maps to replace pixel values for nearby objects. 
+ 
+Sixteen empty regions of varying sizes, with dimensions ranging from 70-700 pixels,  at different positions on the image plane, were visually identified. A $50\times50$ covariance matrix for the pixel values was calculated for each empty region (separately in each filter) and their mean computed, as shown in \autoref{fig:covar}.
+Noise maps generated from the mean covariance matrix for each filter are shown in \autoref{fig:noise_map}.
+
+%----------------------------------
+\begin{figure}[h]
+  \centering
+  \subfloat[Covariance map in V (F606W) filter.]{\includegraphics[width=0.4\textwidth]{covar_f814.png}}
+  \hfill
+  \subfloat[Covariance map in I (F814W) filter.]{\includegraphics[width=0.4\textwidth]{covar_f606.png}}
+  \caption{Mean co-variance matrix in the two filters.}
+  \label{fig:covar}
+\end{figure}
+%----------------------------------
+
+%---------------------------------
+\begin{figure}[h]
+\centering\includegraphics[width=1.\linewidth]{noise_map.png}
+\caption{Image of noise map generated from covariance matrices, for I band (left) and V band (right).}
+\label{fig:noise_map}
+\end{figure}
+%----------------------------------
+
+To replace nearby objects with noise, a region that is identical in size and shape to the nearby object in the segmentation map is cut from a random location in the noise map shown in \autoref{fig:noise_map} and used to replace the pixel values for the nearby object in the galaxy postage stamp.  
+
+\subsection{Parametric Fits}
+The COSMOS catalog galaxies were subject to an additional step of fitting for parametric models. This step was not performed here, and the values in the output catalogs corresponding to these parameters were manually set to 999.
+
+\subsection{PSF Postage Stamp}
+The PSF for each galaxy is taken to be the {\tt Tiny Tim} model PSF image whose location in the {\tt Tiny Tim} grid is closest to that galaxy's position in the focal plane, for the best-fit focus offset described in \autoref{ssec:psf}. Square postage stamps ($20 \times 20$ pixels) are drawn for each filter.
+
+\subsection{Additional Catalogs}
+The Extended Groth Strip (EGS) has been extensively studied with other surveys in other bands. 
+We include in our catalog photometry and photometric redshift measurements from other catalogs, for objects in our catalog with a center within $1''$ of an object in the other catalog.
```

Comment:
I'm pretty sure you have spectroscopic redshifts, not photometric.  If it's DEEP2 then it's spectroscopic (correction needed here and also a few lines below here).